python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -20,18 +20,19 @@ def test_groupby_basic(series, aggregation):
     np.random.seed(0)
     pdf = pd.DataFrame(
         {
-            "x": np.random.randint(0, 5, size=10000),
+            "xx": np.random.randint(0, 5, size=10000),
```

Comment:
I would add a comment to indicate that this is a specially-constructed input where one column name is a substring of the groupby key. Otherwise this special construction will be invisible and overlooked by some later reader who wants to refactor tests.